src/com/ms/silverking/util/jvm/Finalization.java

```diff
@@ -41,7 +42,7 @@ public boolean forceFinalization(long minFinalizationIntervalMillis) {
                     sw.stop();
                     finalized = true;
                     if (verboseFinalization) {
-                        System.out.printf("Finalization completed in %f seconds\n", sw.getElapsedSeconds());
+                        Log.finef("Finalization completed in %f seconds\n", sw.getElapsedSeconds());
```

Comment:
verboseFinalization should control the output, which means that the logging level needs to be a visible level - i.e. warning